tests/smoke/service.bats

```diff
@@ -2,16 +2,17 @@
 
 @test "create" {
   l0 environment create env_name
-  l0 deploy create ./common/Service.Dockerrun.aws.json dpl_name1
-  l0 deploy create ./common/Service.Dockerrun.aws.json dpl_name2
+  l0 deploy create ./common/Service_stateful.Dockerrun.aws.json dpl_name_stateful1
+  l0 deploy create ./common/Service_stateful.Dockerrun.aws.json dpl_name_stateful2
+  l0 deploy create ./common/Service_stateless.Dockerrun.aws.json dpl_name_stateless
   l0 loadbalancer create --port 80:80/http env_name lb_name1
```

Comment:
You'll need the `--type classic` as this will create an `application` type load balancer.

Reply:
✅